datadog_checks_base/datadog_checks/base/utils/common.py

```diff
@@ -5,12 +5,12 @@
 import re
 from decimal import ROUND_HALF_UP, Decimal
 
-from six import PY3
+from six import PY3, string_types
 from six.moves.urllib.parse import urlparse
 
 
 def ensure_bytes(s):
-    if not isinstance(s, bytes):
+    if isinstance(s, string_types) and not isinstance(s, bytes):
```

Comment:
Why this change is necessary ?

Ideally, I would use a strategy similar to https://github.com/pypa/wheel/blob/master/wheel/util.py

Reply:
Because we could try to `ensure_bytes` a non string variable, so we would use `.endcode('utf-8')` which does not exist

Reply:
In that case, the second part `not isinstance(s, bytes)` seems not needed ? 

The second part is always true if the first part is true.